src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -257,8 +271,35 @@ public Instance(
         } else {
             log.info("collect_default_jvm_metrics is false - not collecting default JVM metrics");
         }
+
+        jmxBean = createJmxBean();
     }
 
+    private ObjectName getObjName(String domain,String instance)
+            throws MalformedObjectNameException {
+        return new ObjectName(domain + ":target_instance=" + ObjectName.quote(instance));
+    }
+
+    private JmxfetchTelemetry createJmxBean() {
+        mbs =  ManagementFactory.getPlatformMBeanServer();
+        JmxfetchTelemetry bean = new JmxfetchTelemetry();
+        log.debug("Created jmx bean for instance: " + this.getCheckName());
+
+        try {
+            jmxBeanName = getObjName("JMXFetch" , this.getName());
+            mbs.registerMBean(bean,jmxBeanName);
+            log.debug("Succesfully registered jmx bean for instance: " + this.getCheckName());
+
+        } catch (MalformedObjectNameException | InstanceAlreadyExistsException
+                | MBeanRegistrationException | NotCompliantMBeanException e) {
+            log.debug("Could not register bean for instance: " + this.getCheckName(),e);
```

Comment:
```suggestion
            log.warn("Could not register bean for instance: " + this.getCheckName(),e);
```

lets up the visibility of this error a bit, as far as we know this should never occur, so if it does, its worth investigating more.